java/sage/plugin/CorePluginManager.java

```diff
@@ -2182,14 +2066,7 @@ else if ("FanArt".equals(packages[i].type))
     }
     finally
     {
-      if (stagedRenameWriter != null)
-      {
-        try
-        {
-          stagedRenameWriter.close();
-        }
-        catch (Exception e){}
-      }
+      IOUtils.closeQuietly(stagedRenameWriter);
```

Comment:
stagedRenameWriter is a local variable that's in there from before and will be null (you should remove that line too). That's what is being passed in to close, not the singleton. :)  And you need to call StagedRenameWriter.get().close() so then it also cleans up its own internal reference to the writer and doesn't fail trying to write to a closed stream if you install a second plugin.